http_check/datadog_checks/http_check/http_check.py

```diff
@@ -242,9 +242,12 @@ def send_status_down(loginfo, message):
         try:
             parsed_uri = urlparse(addr)
             self.log.debug("Connecting to %s" % addr)
+
             if disable_ssl_validation and parsed_uri.scheme == "https" and not ignore_ssl_warning:
-                self.warning("Skipping SSL certificate validation for %s based on configuration"
-                             % addr)
+                if 'disable_ssl_validation' in instance:
+                    self.warning('Parameter disable_ssl_validation for {0} is not explicitly set, defaults to true'.format(addr))
+                else:
+                    self.warning("Skipping SSL certificate validation for {0} based on configuration".format(addr))
```

Comment:
You'll need to flip the warnings around.

Reply:
Sorry about that!

Switched em 👍